packages/manager/apps/dedicated/client/app/ip/translations/Messages_fr_FR.json

```diff
@@ -33,18 +33,18 @@
   "ip_alerts_service_other": "{} alertes pour ce service",
   "ip_alerts_spam": "Aucune IP bloquée (Antispam)",
   "ip_alerts_spam_1": "1 IP bloquée (Antispam)",
-  "ip_alerts_spam_help": "Spam (port 25 bloqué suite à une détection d’activité de spam sur l’adresse IP)",
+  "ip_alerts_spam_help": "<strong>Spam</strong> (port 25 bloqué suite à une détection d’activité de spam sur l’adresse IP)",
```

Comment:
**question:** Can we split into 2 sentences and avoid HTML In translation files.